core/src/main/java/org/arquillian/smart/testing/scm/git/GitChangeResolver.java

```diff
@@ -121,24 +142,13 @@ public boolean isApplicable() {
         return allChanges;
     }
 
-    private Set<Change> retrieveCommitsChanges() {
-        try (ObjectReader reader = git.getRepository().newObjectReader()) {
-            final ObjectId oldHead = git.getRepository().resolve(previous + ENSURE_TREE);
-            final ObjectId newHead = git.getRepository().resolve(head + ENSURE_TREE);
-
-            final CanonicalTreeParser oldTree = new CanonicalTreeParser();
-            oldTree.reset(reader, oldHead);
-            final CanonicalTreeParser newTree = new CanonicalTreeParser();
-            newTree.reset(reader, newHead);
-
-            final List<DiffEntry> commitDiffs = git.diff().setNewTree(newTree).setOldTree(oldTree).call();
-            return extract(commitDiffs, repoRoot);
-        } catch (IOException | GitAPIException e) {
-            throw new IllegalStateException(e);
-        }
+    private List<DiffEntry> findRenames(List<DiffEntry> commitDiffs) throws IOException {
```

Comment:
Is it only finding Renamed DiffEntry or it's finding renamed and others(Added, Modified) as well?

Reply:
As I commented on #160 - by default jgit sees it as `DELETE` and `ADD` - this checks if they are actually renames.

Reply:
ok But what about other entries like `Modify` we should consider that as well right?

Reply:
See https://github.com/arquillian/smart-testing/pull/162#discussion_r139666555